plugins/statistics.koplugin/main.lua

```diff
@@ -280,31 +422,95 @@ function ReaderStatistics:getTotalStats()
            total_stats
 end
 
+
+function ReaderStatistics:getAdvStats()
+    local total_stats = {
+        {
+            T(_"Last Week"),"",
+            callback = function()
+                UIManager:show(KeyValuePage:new{
+                    title = T(_"Week Statistics"),
+                    kv_pairs = getDatesFromAll(7,"daily_weekday"),
+                })
+            end,
+        },
+        {
+            T(_"Last Month"),"",
```

Comment:
Same as previous comment, `_("Last month")` should be enough. Please also keep the naming convention consistent since other menu items only capitalized the first word :)